ChangeLog.md

```diff
@@ -11,3 +12,16 @@
 ### Bug fixes
 
 ### Miscellaneous
+
+## Version 0.1.1-dev 
+
+### Features
+
+### Backward-incompatible changes [stable]
+
+### Performance enhancements
+
+### Bug fixes
+Fix total pipeline failure when a sample contains 1 read after dereplcation+singleton removal. Now issues a warning and continues
```

Comment:
Can you please associate the specific PR? The scikit-bio ChangeLog is a great reference, and it makes it trivial for a user to look up the specific changes if necessary